handler/handlers.go

```diff
@@ -49,18 +51,20 @@ type Func func(http.ResponseWriter, *http.Request) error
 type handler struct {
 	proxy   *httputil.ReverseProxy
 	token   ntokend.TokenProvider
+	access  service.AccessProvider
```

Comment:
same as above, maybe `AccessTokenProvider` is better?

Reply:
`AccessTokenProvider`👍🏻
but better to change as a set with `RoleProvider => RoleTokenProvider`.
Refactor in this PR, OR, create another issue and fix it later?

Reply:
I created another issue.
https://github.com/yahoojapan/athenz-client-sidecar/issues/36

In this PR, proceed with current name convention.